apps/sms/test/marionette/lib/views/thread.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+var View = require('./view');
+
+function Thread() {
```

Comment:
As this is linked to the initialization of subpages, let's move this discussion on general thread.